swagger.yaml

```diff
@@ -586,46 +594,20 @@ definitions:
     type: object
     properties:
       vendor:
-        type: string
         description: The name of the device vendor (for example "Intel Corporation")
-      vendor_id:
         type: string
+      vendor_id:
         description: ID of the vendor (for example "8086")
-      device_id:
         type: string
+      device_id:
         description: ID of the device (for example "3ea0")
-      name:
         type: string
+      name:
         description: Product name of the device (for example "UHD Graphics 620 (Whiskey Lake)")
-      address:
         type: string
+      address:
         description: Device address (for example "0000:00:02.0")
-
-  heartbeat:
```

Comment:
the heartbeat is reported by the devices to the control plane.
it shouldn't be removed.